protected/views/site/contact.php

```diff
@@ -13,15 +13,17 @@
 ?>
  <div class="content">
             <div class="container">
-                <section class="page-title-section">
+                <div class="section page-title-section">
                     <div class="page-title">
-                        <ol class="breadcrumb pull-right">
-                            <li><a href="/">Home</a></li>
-                            <li class="active">Contact</li>
-                        </ol>
+                        <nav aria-label="breadcrumbs">
+                            <ol class="breadcrumb pull-right">
+                                <li><a href="/">Home</a></li>
+                                <li class="active">Contact</li>
+                            </ol>
+                        </nav>
                         <h4>Contact</h4>
                     </div>
-                </section>
+                </div>
                 <div class="subsection">
```

Comment:
note: the rest of changes of this file are just result of autoformatting with no code changes